data/api/client-server/content-repo.yaml

```diff
@@ -12,154 +12,164 @@
+              schema:
+                $ref: definitions/errors/rate_limited.yaml
       tags:
         - Media
   "/download/{serverName}/{mediaId}":
     get:
-      summary: "Download content from the content repository."
+      summary: Download content from the content repository.
       operationId: getContent
-      produces: ["*/*"]
       parameters:
         - in: path
-          type: string
           name: serverName
-          x-example: matrix.org
           required: true
           description: |
             The server name from the `mxc://` URI (the authoritory component)
+          example: matrix.org
+          schema:
+            type: string
         - in: path
-          type: string
           name: mediaId
-          x-example: ascERGshawAWawugaAcauga
           required: true
           description: |
             The media ID from the `mxc://` URI (the path component)
+          example: ascERGshawAWawugaAcauga
+          schema:
+            type: string
         - in: query
-          type: boolean
           name: allow_remote
-          x-example: false
           required: false
-          default: true
           description: |
             Indicates to the server that it should not attempt to fetch the media if it is deemed
             remote. This is to prevent routing loops where the server contacts itself. Defaults to
             true if not provided.
+          example: false
+          schema:
+            type: boolean
+            default: true
       responses:
-        200:
-          description: "The content that was previously uploaded."
+        "200":
+          description: The content that was previously uploaded.
           headers:
             Content-Type:
-              description: "The content type of the file that was previously uploaded."
-              type: "string"
+              description: The content type of the file that was previously uploaded.
+              schema:
+                type: string
             Content-Disposition:
-              description: |-
-                The name of the file that was previously uploaded, if set.
-              type: "string"
-          schema:
-            type: file
-            # This is a workaround for us not being able to say the response is required.
-            description: "**Required.** The bytes for the uploaded file."
-        502:
-          description: |-
-            The content is too large for the server to serve.
-          examples:
-            application/json: {
-              "errcode": "M_TOO_LARGE",
-              "error": "Content is too large to serve"
-            }
-          schema:
-            "$ref": "definitions/errors/error.yaml"
-        429:
+              description: The name of the file that was previously uploaded, if set.
+              schema:
+                type: string
+          content:
+            application/octet-stream:
+              schema:
+                type: string
```

Comment:
```suggestion
                type: string
                format: binary
```

Reply:
This has actually changed in the [OpenAPI 3.1 spec](https://spec.openapis.org/oas/v3.1.0#considerations-for-file-uploads).
The spec says to write it like this:

```yaml
application/octet-stream: {}
```

Reply:
Gotta say, I like that new notation; "binary strings" made me cringe when I first saw that...